README.md

```diff
@@ -40,6 +42,7 @@ Example:
 ```groovy
 repositories {
 	maven { url 'https://oss.jfrog.org/oss-snapshot-local' }
+	maven { url 'https://repo.spring.io/milestone' }
```

Comment:
```suggestion
	maven { url 'https://repo.spring.io/snapshot' }
```

We usually depend on the snapshots or reactor, thus if such occasion happens, it is better to specify a snapshot repo

Reply:
Also, this does not fix #940 since this section is related to the snapshots, but M1 now specified as a release one, so I would suggest adding a similar section for release and keep it until we have a GA for 1.1.0